packages/elements/package.json

```diff
@@ -15,7 +15,7 @@
     "@radix-ui/react-use-controllable-state": "^1.2.2",
     "@repo/shadcn-ui": "workspace:*",
     "@xyflow/react": "^12.8.6",
-    "ai": "5.0.72",
+    "ai": "5.1.0-beta.22",
```

Comment:
Tool execution approvals are available starting in: https://github.com/vercel/ai/releases/tag/ai@5.1.0-beta.22